test/fixtures/bpmn/model-complete-semantic.bpmn

```diff
@@ -6,6 +6,7 @@
     <semantic:participant id="participant_1_id" name="Pool 1" processRef="WFP-6-" />
     <semantic:participant id="participant_2_id" processRef="process_2" />
     <semantic:participant id="participant_3_id" name="Black Box Process" />
+    <semantic:participant id="participant_4_id" name="Pool containing sublanes" processRef="process_4" />
```

Comment:
:warning: sill missing tests for the JSon parser about the sublanes, the test here is to ensure the global parsing and the mxgraph model filling is done

Reply:
ok, tests added in 422bfae